editor/import/resource_importer_wav.cpp

```diff
@@ -112,7 +112,15 @@ Error ResourceImporterWAV::import(const String &p_source_file, const String &p_s
 	}
 
 	/* GET FILESIZE */
-	file->get_32(); // filesize
+
+	// The file size in header is 8 bytes less than the actual size.
+	// See https://docs.fileformat.com/audio/wav/
+	const int FILE_SIZE_HEADER_OFFSET = 8;
+	uint32_t file_size_header = file->get_32() + FILE_SIZE_HEADER_OFFSET;
```

Comment:
```suggestion
	const int FILE_SIZE_HEADER_OFFSET = 8;
	uint32_t file_size_header = file->get_32() - FILE_SIZE_HEADER_OFFSET;
```

Reply:
The file size in the header is 8 bytes less than the actual file size. The fist `file->get_32()` reads the file size in the header, and we need to add 8 bytes so that it reflects the real file size. If change it to minus, it will have false warning.